packs/data/feats.db/soaring-armor.json

```
@@ -25,7 +25,14 @@
                 }
             ]
         },
-        "rules": [],
+        "rules": [
+            {
+                "key":"BaseSpeed",
+                "selector":"fly",
+                "predicate":["armor:id:{actor|flags.pf2e.armorInnovationId}"],
+                "value":"@actor.system.attributes.speed.total"
+            }
```

```suggestion
            {
                "key": "BaseSpeed",
                "selector": "fly",
                "predicate": [
                    "armor:id:{actor|flags.pf2e.armorInnovationId}"
                ],
                "value": "@actor.system.attributes.speed.total"
            }
```
Are you editing these REs in the JSON directly? Or are you editing them in Foundry then extracting them? We want them passed through extractpacks to reduce extract noise later and make sure the format is correct.

---

I wrote and tested them on my local instance but no I just copied the data files out since I wasn't making any programmatic changes. Let me run the export tool.